docs/reference/elasticsearch/configuration-reference/remote-clusters.md

```diff
@@ -76,4 +76,77 @@ To use [proxy mode](docs-content://deploy-manage/remote-clusters/remote-clusters
 :   The number of socket connections to open per remote cluster. The default is `18`.
 
 `cluster.remote.<cluster_alias>.server_name`
-:   An optional hostname string which is sent in the `server_name` field of the TLS Server Name Indication extension if [TLS is enabled](docs-content://deploy-manage/security/secure-cluster-communications.md#encrypt-internode-communication). The TLS transport will fail to open remote connections if this field is not a valid hostname as defined by the TLS SNI specification.
\ No newline at end of file
+:   An optional hostname string which is sent in the `server_name` field of the TLS Server Name Indication extension if [TLS is enabled](docs-content://deploy-manage/security/secure-cluster-communications.md#encrypt-internode-communication). The TLS transport will fail to open remote connections if this field is not a valid hostname as defined by the TLS SNI specification.
+
+## Remote cluster strong verification settings [remote-cluster-signing-settings]
+::::{warning}
+This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
+::::
```

Comment:
this is the tagging I am assuming applies given where you added the instructions in docs-content. do these settings apply if one of the clusters is ECE, ECK, or hosted?

we need to indicate what version where the settings are introduced as well. usually we'd do this at the setting level but since you have a heading here we can use that instead

we've been skipping tech preview warning notes because the `preview` applies tag contains the warning. this way, all you need to do is add a new lifecycle state when the feature goes GA. 

<img width="320" height="170" alt="image" src="https://github.com/user-attachments/assets/81a70822-222e-4bdd-9cda-f2c8b6d2192c" />


````suggestion
```{applies_to}
deployment:
  self: preview 9.3
```
````